### ISSUE TITLE
TDM: Open Role for: Software Developer, Accessibility Lead

### ISSUE THREAD

**Issue by ryanfchase** [hackforla/tdm-calculator#2603]
<img src="https://www.hackforla.org/assets/images/projects/tdm-calculator.jpg">

**Project Name**:  

TDM Calculator

**About**

“Transportation Demand Management,” also known as "TDM," is a set of strategies that aim to reduce the number of vehicles on the road and improve the efficiency of transportation systems. One such strategy that the city of Los Angeles employs to accomplish this is requiring real estate developers to meet a set of criteria that help reduce the amount of cars driven by occupants in their buildings.

The Los Angeles TDM Calculator is a web application that Hack for LA is creating in conjunction with the L.A. Department of Transportation (LADOT) and Los Angeles City Planning. The application's purpose is to help real estate developers process their applications more easily and efficiently. It gives real-time feedback while developers explore TDM strategies. It also allows transparency so that the general public can understand which TDM strategies Los Angeles employs.

**Volunteer Opportunity**
Help us design the final features of the Los Angeles TDM web application as well as a new case study website that will document and promote the work Hack for LA did on the TDM app.

We are looking for a lead software engineer to contribute to site accessibility on the TDM application by

- Assessing site accessibility with tools like [WAVE](https://wave.webaim.org/extension/) to generate reports
- Testing site accessibility features with programs like [NonVisual Desktop Access (NVDA)](https://www.nvaccess.org/about-nv-access/)
- Collaborating with developers on the TDM team to achieve site-wide [WCAG compliance](https://www.w3.org/WAI/standards-guidelines/wcag/)
- Communicating accessibility standards and requirements to stakeholders

We expect software developers applying to this position to have a solid understanding of Javascript, HTML, and CSS and a strong interest in advancing accessibility standards for the modern web ecosystem. Nice to haves include familiarity with React and experience with web accessibility tools (e.g. WAVE, NVDA)

**Duration**: The TDM project is slated to last until late October or November.

**Meeting Times:**

Required
Dev Team meeting: Wednesdays 6:00 PM - 7:00 PM Pacific
All-Hands meeting: Wednesdays 7:00 PM - 8:00 PM Pacific

Optional
Stakeholder meeting: Every other Tuesday 4:00 PM - 5:00 PM Pacific
Stakeholder Follow-up meeting: Every other Tuesday 5:00 PM - 6:00 PM Pacific
Stakeholder Preparation meeting: Thursdays 1:00 PM - 2:00 PM Pacific

**Communicate your interest**
Slack channel link: [#tdm-calculator](https://hackforla.slack.com/archives/CKY65G266)
Slack person to contact in the channel: John Darragh, Ryan Chase

**More Information**: Visit Hack for LA's [TDM Calculator project page](https://www.hackforla.org/projects/tdm-calculator).
